### PR TITLE
Specifying that disaster recovery requires a healthy node

### PR DESCRIPTION
**Pull request by bergerhoffer**
Per @hexfusion 

https://issues.redhat.com/browse/ETCD-125

Previews:
* https://deploy-preview-34218--osdocs.netlify.app/openshift-enterprise/latest/backup_and_restore/disaster_recovery/about-disaster-recovery.html
* https://deploy-preview-34218--osdocs.netlify.app/openshift-enterprise/latest/backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.html#dr-scenario-2-restoring-cluster-state_dr-restoring-cluster-state

### CODE REVIEW

File: backup_and_restore/disaster_recovery/about-disaster-recovery.adoc [hunk 1]
```diff
@@ -11,6 +11,11 @@ how to recover from several disaster situations that might occur with their
 more of the following procedures to return your cluster to a working
 state.
 
+[IMPORTANT]
+====
+Disaster recovery requires you to have at least one healthy master host.
```
Comment (hexfusion) [inline backup_and_restore/disaster_recovery/about-disaster-recovery.adoc:16]: s/master/control-plane?

File: modules/dr-restoring-cluster-state.adoc [hunk 2]
```diff
@@ -17,6 +17,7 @@ When you restore your cluster, you must use an etcd backup that was taken from t
 .Prerequisites
 
 * Access to the cluster as a user with the `cluster-admin` role.
+* A healthy master host to use as the recovery host.
```
Comment (hexfusion) [inline modules/dr-restoring-cluster-state.adoc:20]: same

**Comment by hexfusion** [comment]
LGTM

File: backup_and_restore/disaster_recovery/about-disaster-recovery.adoc (hunk 1 above)
Comment (bergerhoffer) [reply backup_and_restore/disaster_recovery/about-disaster-recovery.adoc:16]: We do use "master host" mostly throughout these docs, so I don't want to introduce it in just this change. We do have efforts planned to s/master/control plane throughout the entire doc set coming down the line

**Comment by bergerhoffer** [comment]
@geliu2016 Can you please review?

Previews:
* https://deploy-preview-34218--osdocs.netlify.app/openshift-enterprise/latest/backup_and_restore/disaster_recovery/about-disaster-recovery.html
* https://deploy-preview-34218--osdocs.netlify.app/openshift-enterprise/latest/backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.html#dr-scenario-2-restoring-cluster-state_dr-restoring-cluster-state

**Comment by bergerhoffer** [comment]
@vikram-redhat @sjstout Requesting a merge exception on this PR. It will still need QE overnight tonight, so it won't make freeze. But these are important statements to clarify, for SNO.

**Comment by lpettyjo** [comment]
LGTM

**Comment by geliu2016** [comment]
LGTM, and i think this declaration is necessary, because we have doc bug about this issue: https://bugzilla.redhat.com/show_bug.cgi?id=1860861

**Comment by vikram-redhat** [comment]
QE and Peer review done. Exception granted. Merging.

**Pull request merged**